vpdq/setup.py

```diff
@@ -6,33 +6,55 @@
 import sys
 import subprocess
 from pathlib import Path
+import os
+import logging
 
 DIR = Path(__file__).parent
 read_me = DIR / Path("vpdq/python/README.md")
```

Comment:
This hardcoding "vpdq/..." is very odd. You can't run setup.py directly because of it. You have to run `vpdq-release.py`. I do not understand this decision.

Reply:
I didn't either, and complained about this exact problem at one point.